thehive-charts/thehive/values.yaml

```
@@ -403,93 +479,109 @@ elasticsearch:
         memory: "3584Mi"
         ephemeral-storage: "4Gi"
 
-    # Master nodes heap size
+    # -- Heap size for ElasticSearch master nodes
     heapSize: "2g"
-    # Configure JVM options as env variable
+    # -- Extra environment variables for ElasticSearch master nodes
     extraEnvVars:
       - name: JVM_OPTS
         value: "-Xms2g -Xmx2g -Xmn200m"
 
     persistence:
+      # -- Enable persistent volume for ElasticSearch master nodes
       enabled: true
+      # -- Storage class for ElasticSearch master node persistent volume
       storageClass: ""
+      # -- Size of ElasticSearch master node persistent volume
       size: "10Gi"
 
     networkPolicy:
+      # -- Enable network policy for ElasticSearch master nodes
       enabled: false
 
   data:
-    # Disable data-dedicated nodes
+    # -- Number of data-dedicated node replicas (disabled by default)
     replicaCount: 0
 
   coordinating:
-    # Disable coordination-dedicated nodes
+    # -- Number of coordinating-dedicated node replicas (disabled by default)
     replicaCount: 0
 
   ingest:
-    # Disable ingestion-dedicated nodes
+    # -- Number of ingest-dedicated node replicas (disabled by default)
     replicaCount: 0
 
   metrics:
     image:
+      # -- ElasticSearch metrics exporter image registry
       registry: docker.io
+      # -- ElasticSearch metrics exporter image repository
       repository: bitnamilegacy/elasticsearch-exporter
       tag: 1.9.0-debian-12-r16
 
   volumePermissions:
     image:
+      # -- Volume permissions init container image registry
       registry: docker.io
+      # -- Volume permissions init container image repository
       repository: bitnamilegacy/os-shell
       tag: 12-debian-12-r51
 
   sysctlImage:
+    # -- Sysctl init container image registry
     registry: docker.io
+    # -- Sysctl init container image repository
     repository: bitnamilegacy/os-shell
     tag: 12-debian-12-r51
 
 ############################
 # MinIO subchart variables #
 ############################
 
-# https://github.com/minio/minio/blob/master/helm/minio/values.yaml
 minio:
-  # Use MinIO dependency Helm Chart
-  # You should disable it if you plan to use another s3-compatible object storage for TheHive
+  # -- Enable MinIO dependency subchart deployment (disable if using external S3-compatible storage)
```

```suggestion
  # -- Enable MinIO dependency subchart deployment (disable if using external S3-compatible storage or a filesystem shared between pods)
```
Adding details, as a PV with ReadWriteMany capability could also do the trick to handle TheHive attachment files